pom.xml

```diff
@@ -50,8 +50,8 @@
         <apache.httpclient.version>4.5.2</apache.httpclient.version>
         <bouncycastle.bcpkix.version>1.55</bouncycastle.bcpkix.version>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
-        <jersey.version>2.25</jersey.version>
-        <jetty.version>9.2.19.v20160908</jetty.version>
+        <jersey.version>2.26-b06</jersey.version>
+        <jetty.version>9.4.6.v20170531</jetty.version>
```

Comment:
Thanks, @ewencp .  Do you want separate Maven coordinates for JDK7 vs. JDK8 (`rest-utils-jdk7` and `rest-utils-jdk8`), or is a Maven classifier ok

I'm guessing the side effect of two different builds of `rest-util` is that there will be two builds of `kafka-rest`, the major downstream user of this lib.

So I believe there would be a similar build question for `kafka-rest`.